packages/nx-plugin/src/generators/e2e-project/e2e.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('NxPlugin e2e-project Generator', () => {
         pluginOutputPath: `dist/libs/my-plugin`,
         npmPackageName: '@proj/my-plugin',
       })
-    ).resolves.not.toThrow();
+    ).resolves.toBeDefined();
```

Comment:
this is a bit weird, but `not.toThrow()` syntax starts to fail when generator returns `Promise<GeneratorCallback>`